### ISSUE TITLE
Stream freezes and the playback is not continued when the fragments were skipped

### ISSUE THREAD

**Issue by rwlodkowski** [mangui/flashls#458]
Hi @mangui there is an issue with fragment skipping. When the certain amount of fragments were skipped because of the loading errors but after that the server problems no longer occur and the next fragments are loaded without issues the playback is stopped and won't resume automatically. The user must seek to another part of the video to wake up the player (using play/pause is not working).

This is especially annoying with live streams, where the playback should be continued when the server problems cease.

Here is the screening illustrating this issue with a VOD stream:
https://youtu.be/yxsrBUCkdwM

Full debug logs from the same session as the screening above:
https://www.dropbox.com/s/jo0j9sjhegrksd1/flashls-debug.log?dl=0

Screening for live stream:
https://youtu.be/7_SVIJQHv8k

Debug logs:
https://www.dropbox.com/s/gjyvdw87sj3u6nc/flashls-debug-live.log?dl=0

**Comment by mangui** [comment]
still happening with v0.4.4.16 ?

**Comment by rwlodkowski** [comment]
Yep it's still there with v0.4.4.16. Screenings and logs are from this version of flashls.

**Comment by rwlodkowski** [comment]
The worst case is with live streams (as you can see in the attached screening) when the playback is stalled and even seeking won't help. The player must be restarted to continue.

**Comment by rwlodkowski** [comment]
@mangui any findings in this case? Cause this making fragment skipping functionality quite useless.

**Comment by mangui** [comment]
Didnt check yet.

**Comment by pratikpradhan** [comment]
We are also facing the same issue with v0.4.4.17.
Any updates to mitigate this.
